csv_joiner.py

```diff
@@ -0,0 +1,147 @@
+import sys
+
+
+class InvalidModeError(Exception):
+    def __init__(self, modes):
+        self.expected = '/'.join(modes)
+        self.message = 'Invalid join mode'
+
+    def __str__(self):
+        return f'{self.message}. Expected: {self.expected}'
+
+
+class InvalidColumnError(Exception):
+    def __init__(self, col):
+        self.col = col
+        self.message = 'Column "{0}" does not appear in one/both files'
```

Comment:
Might be useful to tell which table is missing this column - just to help the user not search it manually